main.js

```diff
@@ -1,103 +1,117 @@
-createLoginForm();
+let validationErrors = {};
+let formValues = {};
+let touchedElements = [];
 
 const userInfo = {
     username: "user",
     password: "test",
     token: ""
 };
 
-const loginForm = document.querySelector("#login")
+createLoginForm();
+
+const loginForm = document.querySelector("#login");
 const usernameInput = document.querySelector("#username");
 const userPassword = document.querySelector("#password");
+const submitButton = document.querySelector("#login_btn");
```

Comment:
Remove all this variables. You don't need them anymore

```
const loginForm = document.querySelector("#login");
const usernameInput = document.querySelector("#username");
const userPassword = document.querySelector("#password");
const submitButton = document.querySelector("#login_btn");
```

Reply:
Done